### ISSUE TITLE
Set learning rate at specify layer

### ISSUE THREAD

**Issue by eric612** [pjreddie/darknet#1200]
I have a question , if I set the learning rate each layer , like below , is it effective ? 

[convolutional]
batch_normalize=1
filters=64
size=3
stride=2
pad=1
activation=leaky
learning_rate=0.0001

[convolutional]
batch_normalize=1
filters=32
size=1
stride=1
pad=1
activation=leaky
learning_rate=0.0001
